src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/DefaultJsonTypeInfoResolver.Helpers.cs

```diff
@@ -526,12 +526,21 @@ private static NullabilityState DetermineParameterNullability(ParameterInfo para
 
                 static byte[]? GetNullableFlags(MemberInfo member)
                 {
-                    foreach (Attribute attr in member.GetCustomAttributes())
+                    foreach (CustomAttributeData attr in member.GetCustomAttributesData())
```

Comment:
How does this change relate to the rest?

Reply:
It isn't, but I decided to fold the change in following the feedback from https://github.com/dotnet/extensions/pull/5591#issuecomment-2450870996